assemble.py

```diff
@@ -158,6 +167,18 @@ def copy_restorable_apps_to_wic(target: FactoryClient.Target, wic_image: str, to
     p.tick()
 
 
+def fetch_apps(target: FactoryClient.Target, token: str, apps_shortlist: list, fetch_dir: str) -> int:
```

Comment:
i'm not seeing what calls this new function?

Reply:
I am not seeing it neither :), so let's remove it.